packages/core/src/index.ts

```diff
@@ -115,16 +117,24 @@ class Hopr<Chain extends HoprCoreConnector> extends EventEmitter {
   private strategy: ChannelStrategy
   private networkPeers: NetworkPeers
   private heartbeat: Heartbeat
+  private logger: Logger
 
   /**
    * @constructor
    *
    * @param _options
    * @param provider
    */
-  private constructor(options: HoprOptions, public _libp2p: LibP2P, public db: LevelUp, public paymentChannels: Chain) {
+  private constructor(
+    options: HoprOptions & { logger: Logger },
```

Comment:
Why is this not part of HoprOptions?

Reply:
Done with https://github.com/hoprnet/hoprnet/pull/1285/commits/99fe64a0368717b8b8e97df79a139ba815792c48